packages/mobile/src/utils/time.ts

```diff
@@ -341,3 +342,12 @@ function quickFormat(timestamp: number, i18next: i18nType, formatRule: string) {
     locale: locales[i18next?.language]?.dateFns ?? locales['en-US']?.dateFns,
   })
 }
+
+function toUtcDate(date: Date) {
+  return new Date(date.getTime() - date.getTimezoneOffset() * 60 * 1000)
+}
+
+export function getNextMondayAt8pmUtc() {
+  const mondayDate = startOfWeek(Date.now(), { weekStartsOn: 1 })
```

Comment:
Sure this actually returns next Monday?

From the documentation example:
```js
// If the week starts on Monday, the start of the week for 2 September 2014 11:55:00:
var result = startOfWeek(new Date(2014, 8, 2, 11, 55, 0), { weekStartsOn: 1 })
//=> Mon Sep 01 2014 00:00:00
```
https://date-fns.org/v2.18.0/docs/startOfWeek

Reply:
This is the previous Monday at 0:00 GMT and we're adding 7 days and 20 hours in the following line, right?

Reply:
yes my bad, all good 👍